packages/cpp/src/browser/cpp-keybinding.ts

```diff
@@ -31,15 +31,15 @@ export class CppKeybindingContribution implements KeybindingContribution {
         @inject(CppKeybindingContext) protected readonly cppKeybindingContext: CppKeybindingContext
     ) { }
 
-    registerKeybindings(registry: KeybindingRegistry): void {
+    registerDefaultKeybindings(registry: KeybindingRegistry): void {
         [
             {
                 commandId: 'switch_source_header',
                 context: this.cppKeybindingContext,
-                keyCode: KeyCode.createKeyCode({ first: Key.KEY_O, modifiers: [Modifier.M3] })
+                keyCode: TheiaKeyCodeUtils.createKeyCode({ first: Key.KEY_O, modifiers: [Modifier.M3] })
```

Comment:
I wonder if renaming `registerKeybinding` -> `registerDefaultKeybindings` is really an improvement, now that keymaps can only be set at once. Also `KeyCode`->`TheiaKeyCodeUtils` is not better IMHO. The whole PR would be much smaller if your reverse those renames.

Reply:
Ok for the `registerKeybinding`. I think I changed it to TheiaKeyCodeUtils as I ended up with a lot of helper functions in the KeyCode class to make the easy naming keys work... I can revert it back also.